typeDefs.js

```diff
@@ -83,6 +93,41 @@ const typeDefs = gql`
     name: [Language]
     images: ImageSet
     available: Boolean
+    simpleName: String
+  }
+
+  type ProductOptions {
+    size: Size
+    bottomMaterial: BottomMaterial
+    bottomColor: String
+    availableCount: Boolean
+    additions: [ProductAdditions]
+  }
+
+  type ProductAdditions {
+    name: [Language]
+    description: [Language]
+    available: Boolean
+    additionalPrice: Int
+  }
+
+  type Size {
```

Comment:
Why is to called `Size` if it contains not only properties related to size

Reply:
it is a collection "Size" in a database that contains all sizes for the product and additional prise for it. It can be renamed but I think this question is more related to database